ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml

```diff
@@ -18,9 +18,8 @@ resources:
       memory: 250Mi
 tests:
 - as: e2e
-  skip_if_only_changed: ^doc/|\.md$|\.adoc$|^(?:.*/)?(\.gitignore|\.codecov\.ya?ml|\.yamllint|sonar-project\.properties|OWNERS|PROJECT|LICENSE)$
```

Comment:
Let's drop this - it's not really useful and it brings more complications than benefits